src/site/content/en/_redirects.yaml

```diff
@@ -1,3 +1,16 @@
+# Master redirects file for web.dev. This mirrors the old DevSite file.
+# There are two types of redirects:
+#
+#  (1) Simple redirects, e.g., "from: /foo"
+#      - These don't match anything under this URL, like "/foo/bar"
+#      - No trailing slash is required, as this matches "/foo", "/foo/" and
+#        "/foo/index.html"
+#
+#  (2) Group redirects, e.g., "from: /foo/...", which will match the top-level
+#      path (with the same rules as (1)) as well as any descendant paths
+#
+# WARNING! Redirects DO NOT take priority over static content.
```

Comment:
Can you add an example so folks can quickly grok what this means.

Reply:
Like, we're trying to say "If you try to redirect /foo, but there's a file at /foo/index.html, then we're going to still serve /foo/index.html. You have to move the file."